client/src/modules/fiscal/fiscal.js

```diff
@@ -25,17 +25,26 @@ function FiscalController($state, Fiscal, ModalService, Notify, $window) {
   vm.sortByDateCreated = 'ASC';
   vm.sortByNumberOfMonth = 'ASC';
 
+  vm.loadingState = true;
+  vm.loadingError = false;
+
+
   // refresh Fiscal Year
   function refreshFiscalYear() {
-    return Fiscal.read(null,{ detailed: 1 })
+    return Fiscal.read(null, { detailed : 1 })
       .then(function (fiscalYears) {
         vm.fiscalYears = fiscalYears;
+      }).catch(() => {
+        vm.loadingError = true;
+        Notify.handleError();
```

Comment:
Notify expects to be passed an error object.  You should use the error that is returned from the server.  Like this:
```js
.catch(function (err) {
  vm.loadingError = true;
  Notify.handleError(err);
})
```